src/index.js

```diff
@@ -23,6 +23,32 @@ const moduleName =
 	moduleAliases[packageName] ||
 	packageName.replace('resin-', '').replace('balena-', '');
 
+function getCodeowners() {
+	// git ls-files '*CODEOWNERS'
+	// * @user1 @user2
+	let codeownerSet = new Set();
+	const codeownersFiles = gitMultilineResults('ls-files *CODEOWNERS');
+	codeownersFiles.forEach(function(file) {
+		codeownerLine = fs
+			.readFileSync(file)
+			.toString()
+			.toLowerCase()
+			.split('\n')
+			.filter(n => n)
+			.filter(n => !n.startsWith('#'));
+		codeownerLine.forEach(function(owner) {
+			codeownerSet = new Set([
+				...owner
+					.split(' ')
```

Comment:
How about doing a regex like `/(@\S+)/`?

Reply:
According to Github's [CODEOWNER spec](https://help.github.com/en/articles/about-code-owners), one can also use email addresses in lieu of usernames. I will update this snippet to use a looser regex